returns/contrib/hypothesis/laws.py

```diff
@@ -28,13 +28,15 @@ class _Settings(NamedTuple):
 
     settings_kwargs: Dict[str, Any]
     use_init: bool
+    mark_xfail: bool
```

Comment:
I really don't want to add this flag, because I don't see any reasons why anyone would need it. Do you have some use-cases in mind?

Reply:
No, it's because I didn't read yet about pytest module marker!! I'll revert it